src/EpisodeIntegrationService/ReceiveData/ReceiveData.cs

```diff
@@ -179,13 +191,30 @@ private async Task ProcessParticipantDataAsync(IEnumerable<BssSubject> subjects,
                 _logger.LogInformation("Sending participant to {Url}: {Request}", participantUrl, serializedParticipant);
 
                 await _httpRequestService.SendPost(participantUrl, serializedParticipant);
+
+                successCount++;
+                rowIndex++;
+                _logger.LogInformation("Row of No.{rowIndex} processed successfully",rowIndex);
             }
         }
+
         catch (Exception ex)
         {
             _logger.LogError(ex, "Error in ProcessParticipantDataAsync: {Message}", ex.Message);
-            await ProcessParticipantDataAsync(subjects, participantUrl);
+
+            failureCount++;
+            rowIndex++;
+            _logger.LogInformation("Row of No.{rowIndex} processed unsuccessfully",rowIndex);
+            await ProcessParticipantDataAsync(name,subjects, participantUrl);
```

Comment:
I think this method will need to be modified so that it no longer calls itself recursively and instead catches exceptions inside the for each loop. Otherwise aren't successCount, failureCount and rowIndex going to be reset to 0 after each bad row is encountered?